common/rpc/params.go

```diff
@@ -84,6 +85,15 @@ func NewParams(serviceName string, config *config.Config, dc *dynamicconfig.Coll
 		return Params{}, fmt.Errorf("public client outbound: %v", err)
 	}
 
+	forwardingRules, err := getForwardingRules(dc)
```

Comment:
fail closed? I think this seems reasonable, but a little risky, do we have a log?